model/src/main/kotlin/utils/DependencyGraphBuilder.kt

```diff
@@ -19,6 +19,8 @@
 
 package org.ossreviewtoolkit.model.utils
 
+import java.util.LinkedList
```

Comment:
In the commit message, I find it a bit misleading to refer to `visitEach()` which did not exist before. Also IMO the main point is not an "out factoring" of code, but the conversion of a recursive algorithm into a queue-based one. So, maybe better say: "DependencyGraphBuilder: Avoid recursion when constructing the graph".